kedro/framework/cli/pipeline.py

```diff
@@ -263,6 +268,9 @@ def pull_package(  # pylint:disable=unused-argument, too-many-arguments
     click.secho(message, fg="green")
 
 
+micropkg.add_command(pull_package)
```

Comment:
I don't know how well that works, given we have `@command_with_verbosity(pipeline, "pull")` around the pull command for instance? Can you check what happens if you run `kedro micropkg package -V` for instance?

Reply:
`-V `is not an option for this command, there is a `-h` option that prints the other options available for `kedro micropkg package`